client/lib/media/utils/create-transient-media.js

```diff
@@ -6,11 +6,9 @@ import path from 'path';
 /**
  * Internal dependencies
  */
-import impureLodash from 'lib/impure-lodash';
 import { getFileExtension } from 'lib/media/utils/get-file-extension';
 import { getMimeType } from 'lib/media/utils/get-mime-type';
-
-const { uniqueId } = impureLodash;
+import createTransientMediaId from 'lib/media/utils/create-transient-media-id';
```

Comment:
I wonder if it's even better if we re-export `createTransientMediaId ` in `lib/media/utils/index.js` and import it from `lib/media/utils` instead, like we do with other of those media utils?

Reply:
good point! let's stay consistent with how we consume `lib/media/utils` 😀